package.json

```diff
@@ -16,33 +21,32 @@
     "clean": "node ./scripts/clean.js",
     "update": "node ./scripts/update.js",
     "reset": "npm run clean && npm ci && npm run update",
-    "postinstall": "npm run bootstrap",
     "prepublishOnly": "npm run build && npm run test"
   },
   "devDependencies": {
-    "@babel/cli": "^7.18.10",
-    "@babel/core": "^7.18.13",
-    "@babel/eslint-parser": "^7.18.9",
-    "@babel/preset-env": "^7.18.10",
-    "@babel/register": "^7.18.9",
+    "@babel/cli": "^7.24.7",
+    "@babel/core": "^7.24.7",
+    "@babel/eslint-parser": "^7.24.7",
+    "@babel/preset-env": "^7.24.7",
+    "@babel/register": "^7.24.6",
     "@rollup/plugin-babel": "^5.3.1",
     "@rollup/plugin-commonjs": "^22.0.2",
     "@rollup/plugin-node-resolve": "^13.3.0",
-    "acorn": "^8.8.0",
-    "ava": "^4.3.1",
+    "acorn": "^8.12.1",
+    "ava": "^1.4.1",
```

Comment:
had to downgrade ava to support peerDependency of ninos, would be good to migrate off ava/ninos in the future, since ninos is no longer maintained